app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -1823,14 +1899,23 @@ object CliCommand {
   case object GetStakingAddress extends CliCommand
   case object ListEvents extends CliCommand
 
-  case class GetEvent(announcementTLV: OracleAnnouncementTLV) extends CliCommand
+  case class GetEvent(eventName: String) extends CliCommand
 
   case class CreateEnumEvent(
       label: String,
       maturationTime: Instant,
       outcomes: Seq[String])
       extends CliCommand
 
+  case class CreateNumericEvent(
+      eventName: String,
+      maturationTime: Instant,
+      maxValue: Long,
+      isSigned: Boolean,
```

Comment:
so either we need to get rid of `isSigned` (we don't have `minValue`, so it has to be zero), or add `minValue` so that we can do signed digit decomp events right?

Reply:
imo I think minValue will make it more confusing, someone might try to do something like `min=1,000 max=10,000` and we'd have to make an event from 0-10,000. Or someone could do `min=-1,000 max=10,000` and wehave have to do -10,000 - 10,000.

Maybe changing `isSigned` to `includeNegatives` would be a better framing?

Reply:
Doesn't the same principle apply with `maxValue` alone? I.e. if I my application requires `maxValue=2^32+1`, at the protocol level `maxValue=2^33`.

This PR is about providing a simple API, which allowing a user to specify their `minValue` will do in my opinion.

Reply:
@benthecarman I agree with your assessment of the consequences of doing it Chris's way but I actually don't think this is a problem, there is no issue with allowing out of bounds things into our domain so long as we cover the entire range from min to max. If we're going for an easy UI we should do this simplification and more sophisticated users can use numdigits is signed etc.

Reply:
Went ahaead with the minValue param